de-lib/src/main/java/org/iplantc/de/theme/base/client/diskResource/grid/cells/DiskResourceDotMenuDisplayStrings.properties

```diff
@@ -0,0 +1,2 @@
+dataLinkText=Create / View Public Link
```

Comment:
Tiny, but it's probably most accurate to say "Public Links", since the dialog it brings up will let you create several even for one file.

Reply:
At least in prod, this brings up the smaller dialog with just a single public link that you can copy and paste somewhere else, not the view where you can choose to create or destroy public links.

Reply:
Oh, you're right. I forgot about that dialog, I think, heh.